packages/terra-status-view/tests/wdio/status-view-spec.js

```diff
@@ -40,4 +40,12 @@ Terra.describeViewports('StatusView', ['medium'], () => {
     browser.url('/raw/tests/terra-status-view/status-view/status-view-thin-constraint');
     Terra.validates.element('error too thin for glyph');
   });
+  it('Overflows correctly when the container height is constrained', () => {
+    browser.url('/raw/tests/terra-status-view/status-view/status-view-overflow');
+    // eslint-disable-next-line prefer-arrow-callback
+    browser.execute(function scrollOptionIntoView() {
```

Comment:
can we use wdio's hook instead? https://v6.webdriver.io/docs/api/element/scrollIntoView.html

also, i'm curious what happens if we don't have this, is the component initially scrolled to the top?

Reply:
Updated here https://github.com/cerner/terra-core/pull/3392/commits/b4f4e487f42787a0c74eb96565a4e1df7e99a710